swagger-spec/nodes/view_only_link_detail.yaml

```diff
@@ -1 +1,42 @@
 # /nodes/{node_id}/view_only_links/{link_id}/
+get:
+  summary: Retrieve a view only link
+  description: >-
+    Retrieves the details of a view only link on a node.
+
+    ####Returns
```

Comment:
I think some of the other View Only Links endpoints have a Permissions section? Can you copy that here too?